pkg/apis/servicecatalog/validation/serviceclass_test.go

```diff
@@ -117,51 +101,6 @@ func TestValidateServiceClass(t *testing.T) {
 			}(),
 			valid: false,
 		},
-		{
```

Comment:
where did these tests go?  Are they no longer valid? Seems like this PR shouldn't have changed the need for them.

Reply:
service classes don't have plans, so yes, it does remove the need for them.

Reply:
unless I'm missing it, these are all testing plans (names, IDs, descriptions,etc...). Moving plans out from under services should not mean we shouldn't test these things any more.  They should just appear some place else in the code.

Reply:
I'll put a validation test in.